integration_test/testutil_test.go

```diff
@@ -84,6 +84,7 @@ func setupKrewBin(t *testing.T, tempDir *testutil.TempDir) string {
 	if err := os.MkdirAll(binPath, 0755); err != nil {
 		t.Fatal(err)
 	}
+	tempDir.Write("receipts/krew.notyaml", []byte("must be present for receipts migration check"))
```

Comment:
This is a bit hacky. But the new logic requires that _something_ is present under `receipts/`.